android/build.gradle

```diff
@@ -10,6 +10,7 @@ buildscript {
         androidXBrowser = "1.4.0"
         DocumentScanner_compileSdkVersion = 33
         DocumentScanner_targetSdkVersion = 33
+        kotlinVersion = '1.6.10' // react-native-receive-sharing-intent requires a global kotlin version else the build fails
```

Comment:
how do I check that I have the same version installed locally?

Reply:
it is my understanding that gradle should get you the correct version on build